elenacaseyroby/YoutubeMusicLib
# Create listens page

# What
- Added "listens" page that lets you view recent listens and update video data

To do:
- Add ability to update Play, Library or Release Date
- Auto-update rows with the same youtube_id when a textfield is updated
# Why

So users can view most recent plays and update incorrect youtube data
# Test
1. Go to http://localhost:5000/listens?
2. Update any field except, Play, Library or Release Date.  If multiple rows contain info for the same video, you must update all or none of them for the updates to go through for that vid.
3. Submit
4. Refresh the page so that the table is repopulated from the db. Your submissions should be reflected.
5. Merge!
